.travis.yml

```diff
@@ -11,8 +11,9 @@ android:
     - extra-google-m2repository
     - addon-google_apis-google-16
 before_script:
-  - cd actionlist && npm install
+  - cd actionlist && rm -rf node_modules && npm install
```

Comment:
Both the pull requests have the same problem with the build and I guess all the further builds will have it too. I am not able to figure out the root cause to this.